sass/components/VmDetails/boot-order.scss

```diff
@@ -0,0 +1,4 @@
+.kubevirt-boot-order__ol {
```

Comment:
I think that its incorrect to use html tag `ol` in css selector. Correct would be something like `.kubevirt-boot-order__list`

Reply:
Ah... I wasn't aware of that. Thanks! Done!